docs/contributing.rst

```diff
@@ -246,9 +245,9 @@ notes (``docs/release.rst``).
 
 The documentation can be built locally by running::
 
-    $ cd docs
-    $ make clean; make html
-    $ open _build/html/index.html
+    cd docs
+    make clean
+    make html
```

Comment:
```suggestion
    make html
    open _build/html/index.html #macos
    xdg-open _build/html/index.html #linux
```

Maybe something like this?

Reply:
Again, I it's best to leave out so there's code that works on all operating systems that you can just copy/paste.